libby/form/ListBox.lib.tsx

```diff
@@ -116,14 +116,14 @@ describe('ListBox', () => {
     function Example() {
       const ref = React.useRef();
```

Comment:
You can do `const ref = React.useRef<HTMLInputElement>();` here instead of specifying `React.MutableRefObject`